README.md

```diff
@@ -45,12 +45,18 @@ fn main() {
 }
 ```
 
-## Feature flags ##
+## Usage in constants ##
+`memoffset` has support for compile-time `offset_of!` on rust>=1.65, or on older nightly compilers.
 
-### Usage in constants ###
-`memoffset` has **experimental** support for compile-time `offset_of!` on a nightly compiler.
+Stable support for const evaluation is not as complete as unstable due to dependence on [`#![feature(const_refs_to_cell)]`](https://github.com/rust-lang/rust/issues/80384).
+This means that if need to get the offset of a cell, you'll have to remain on nightly for now.
```

Comment:
I feel like this should be 3 sections:
- recent stable
- recent nightly (to get Cell)
- old nightly